include/fiber_backends.h

```diff
@@ -59,6 +61,7 @@ const struct fiber_map_backend_d2z fiber_execute_d2z[] = {
     { "nb3dfft",    &compute_d2z_nb3dfft   },
     { "fftw",       &compute_d2z_fftw      },
     { "fftadvmpi",  &compute_d2z_fftadvmpi }
```

Comment:
Looks like some "," are missing here, above and below.

Reply:
Thanks, Stan,  fixed it!